Typescript example not working when importing files

If in typescript example, you try to import function from a different file like
```
import { getMessage } from './message';

export async function onMessage(
  originString: string,
  requestObject: Record<string, unknown>,
) {
  switch (requestObject.method) {
    case 'hello':
      return wallet.request({
        method: 'snap_notify',
        params: [
          {
            type: 'native',
            message: getMessage(originString),
          },
        ],
      });
    default:
      throw new Error('Method not found.');
  }
}

```

Build fails with error:
```
Error: Can't walk dependency graph: Cannot find module './message' from '/Projects/SNAPS/snaps-skunkworks/packages/examples/examples/typescript/src/index.ts'
    required by /Projects/SNAPS/snaps-skunkworks/packages/examples/examples/typescript/src/index.ts
    at /Projects/SNAPS/snaps-skunkworks/node_modules/resolve/lib/async.js:137:35
    at load (/Projects/SNAPS/snaps-skunkworks/node_modules/resolve/lib/async.js:156:43)
    at onex (/Projects/SNAPS/snaps-skunkworks/node_modules/resolve/lib/async.js:181:17)
    at /Projects/SNAPS/snaps-skunkworks/node_modules/resolve/lib/async.js:15:69
    at FSReqCallback.oncomplete (node:fs:198:21) {
```

Also, typescript example is missing snaps-cli dependency^^

--- a/package.json
+++ b/package.json
@@ -23,10 +23,10 @@
     "build": "yarn build:pre-tsc && yarn build:tsc && yarn build:post-tsc",
     "build:clean": "yarn clean && yarn build",
     "build:tsc": "tsc --build",
-    "build:pre-tsc": "yarn workspaces foreach --parallel --topological run build:pre-tsc",
-    "build:post-tsc": "yarn workspaces foreach --parallel --topological run build:post-tsc",
-    "clean": "yarn workspaces foreach --parallel run clean",
-    "test": "yarn workspaces foreach --parallel run test",
+    "build:pre-tsc": "yarn workspaces foreach --parallel --topological -v run build:pre-tsc",
+    "build:post-tsc": "yarn workspaces foreach --parallel --topological -v run build:post-tsc",
+    "clean": "yarn workspaces foreach --parallel -v run clean",
+    "test": "yarn workspaces foreach --parallel -v run test",
     "test:ci": "yarn workspaces foreach --parallel run test:ci"
   },
   "simple-git-hooks": {

--- a/packages/cli/src/cmds/build/bundle.ts
+++ b/packages/cli/src/cmds/build/bundle.ts
@@ -29,6 +29,7 @@ export function bundle(
   return new Promise((resolve, _reject) => {
     const bundler = browserify(src, {
       debug,
+      extensions: ['.js', '.ts'],
       // Standalone is required to properly support Snaps using module.exports
       standalone: '<snap>',
     });

--- a/packages/examples/examples/typescript/package.json
+++ b/packages/examples/examples/typescript/package.json
@@ -32,6 +32,7 @@
     "@metamask/eslint-config-jest": "^8.0.0",
     "@metamask/eslint-config-nodejs": "^8.0.0",
     "@metamask/eslint-config-typescript": "^8.0.0",
+    "@metamask/snaps-cli": "workspace:*",
     "eslint": "^7.30.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.23.4",

--- a/packages/examples/examples/typescript/snap.manifest.json
+++ b/packages/examples/examples/typescript/snap.manifest.json
@@ -7,7 +7,7 @@
     "url": "https://github.com/MetaMask/snap-template.git"
   },
   "source": {
-    "shasum": "FNIh+AoDsE1/jPlbSH/XeyEBWNCNxUJt/eQU/l7gGnQ=",
+    "shasum": "6lr8vy2VxV31PDNNnYlDfigBARuIkr5fwH+UHIWOyLU=",
     "location": {
       "npm": {
         "filePath": "dist/bundle.js",

--- a/packages/examples/examples/typescript/src/index.ts
+++ b/packages/examples/examples/typescript/src/index.ts
@@ -1,4 +1,4 @@
-const getMessage = (originString: string): string => `Hello, ${originString}!`;
+import { getMessage } from './message';
 
 export async function onMessage(
   originString: string,

--- a/packages/examples/examples/typescript/src/message.ts
+++ b/packages/examples/examples/typescript/src/message.ts
@@ -1,0 +1,2 @@
+export const getMessage = (originString: string): string =>
+  `Hello, ${originString}!`;

